tomcat/src/main/docker/50-tomcat.bash

```diff
@@ -3,4 +3,13 @@
 # If we are deploying on a GAE platform, copy gcp configuration
 if [ "$PLATFORM" == "gae" ]; then
   cp /config/gcp.xml ${CATALINA_BASE}/conf/gcp.xml
-fi
\ No newline at end of file
+fi
+
+if [ "$ENABLE_DISTRIBUTED_SESSIONS" == "true" ]; then
```

Comment:
I think we should instead use something like `TOMCAT_MODULES_ENABLE=gcp-datastore-sessions` to be more compatible with how Jetty does it.